model/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "model"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
+bdk = { version = "0.16", default-features = false }
+cached = { version = "0.30.0", default-features = false, features = ["proc_macro"] }
+chrono = { version = "0.4", features = ["serde"] }
+hex = "0.4"
+itertools = "0.10"
+maia = "0.1.0"
+nalgebra = { version = "0.30", default-features = false, features = ["std"] }
+ndarray = "0.15.4"
+ndarray_einsum_beta = "0.7.0"
+num = "0.4.0"
+parse-display = "0.5.3"
+reqwest = { version = "0.11", default-features = false }
+rust_decimal = "1.21"
+rust_decimal_macros = "1.21"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+serde_with = { version = "1", features = ["macros"] }
+sqlx = { version = "0.5", features = ["sqlite", "runtime-tokio-rustls"] }
```

Comment:
Couple of open questions regarding this:

- Should we trim the feature list here? Does that negatively (poisoned cache) or positively (less code) influence compile times?
- Can/Should we perhaps get rid of it entirely by mapping on the way out of the DB layer?